wagn/rails/controllers/card_controller.rb

```diff
@@ -248,7 +243,9 @@ def show view = nil, status = 200
     else
       args = { :text=>result, :status=>status }
       args[:content_type] = 'text/text' if format == :file
-      render args
+      card.run_callbacks :render_view do
```

Comment:
render_page? but for an ajax request "page" is not the best description. Maybe just "render" because that's the point where all the view rendering starts?